charts/hedgedoc/templates/deployment.yaml

```
@@ -27,6 +27,10 @@ spec:
       serviceAccountName: {{ include "hedgedoc.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      volumes:
+        - name: uploads
+          persistentVolumeClaim:
+            claimName: {{ template "hedgedoc.fullname" . }}
```

i'm semi-sure that it'll also need something like this
```suggestion
      {{ if .Values.hedgedoc.persistentStorage.enabled }}
      volumes:
        - name: uploads
          persistentVolumeClaim:
            claimName: {{ template "hedgedoc.fullname" . }}
      {{- end }}
```